tests/cli/examples/single_workflow.py

```diff
@@ -0,0 +1,3 @@
+from hera.workflows import Workflow
+
+workflow = Workflow(name="single")
```

Comment:
We also need tests/examples for `WorkflowTemplate` and `ClusterWorkflowTemplates`

Reply:
Given that they both inherit from `Workflow`, I wouldn't think it would matter? So long as they both have tested implementations of `to_yaml`, it seems like they'd exhibit identical behaviors.

I can certainly add tests for them, or else parametrize the tests on each of the class types. I just dont know what would be the differentiating factor of the tests, if they existed otherwise.

---

For reference, the `examples/` folder thing is solely so that I could fully exercise the importlib shenans used for loading the python files against proper modules.

Reply:
True, but it's more so that users can see it's not just for Workflows - helps avoid repeating simple questions!